app/waf/src/repositories/common.ts

```diff
@@ -22,4 +31,42 @@ export class CommonRepository<
 
     return super.updateById(id, data, options);
   }
+
+  findEntitiesByTenant(tenantId: string): Promise<T[]> {
+    const filter: Filter = {where: {tenantId: tenantId}};
+    // maybe remove this "magic conversion" someday.
+    return this.find(filter as (Filter<T>));
+  }
+
+  async findEntityByTenant(tenantId: string, id: ID): Promise<T> {
```

Comment:
Just thinking that the name `findEntitiesByTenant` and `findEntityByTenant` is so similiar. 

How about 
`findEntitiesByTenant` => `filterByTenant`, returns multiple ones. 
`findEntityByTenant` => `findByTenant`(The style is very like `findById` which returns only one entity.)

`Entity` in the name is not needed, is it? because the class name is just **Repository** which operates on `Entity`.

Reply:
hmm..., do you think filterByTenant is really clear in expression?

Reply:
Seen from existing operations of Repository, they are `update`, `find`, `findById`, and `updateById`, there's no entity there. So I think it's better to myself thinking.